main.go

```diff
@@ -95,6 +95,18 @@ func registerCommand(c command.Command) {
 	helpData[c.Category][c.Name] = c
 }
 
+func registerHardcodedCommand(name, helpText, category, response string) {
```

Comment:
Can we move this logic into `registerLinkCommand` as it only is used there?

Reply:
Done.